compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -765,6 +765,10 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             _ => None,
         };
 
+        if let Some(intrinsic @ (sym::copy_nonoverlapping | sym::transmute)) = intrinsic {
+            bug!("Intrinsic {intrinsic} should have been lowered before codegen");
+        }
```

Comment:
nit: Remove the check. Maintaining a list of lowered intrinsics seems counter-productive. This code path would eventually emit an "unknown intrinsic" error anyway.